src/utils/emailTemplates/buildUserInvitedEmailBody.js

```diff
@@ -0,0 +1,50 @@
+const buildUserInvitedEmailBody = (email, experimentId, inviterUser) => {
+  const link = `scp.biomage.net/experiments/${experimentId}/data-processing`;
+
+  const messageToSend = `
```

Comment:
I think that now that we will have 2 templates it's time to move into something more robust. 
Ideally, we'd move into https://www.npmjs.com/package/email-templates which handles all the email parameters, rendering & sending in one place. 

If you feel that's out of the scope, I would migrate at least the templates into the `pug` template system (which is used by the `email-templates` package) 
* Package: https://pugjs.org/api/getting-started.html
* Guide: https://www.sitepoint.com/a-beginners-guide-to-pug/

Using PUG we would render the templates just doing:
```
pug.renderFile('inviteUserTemplate.pug', {
  inviter: inviterUser,
  experimentId: experimentId,
})
```